internal/service/ldapconfiguration/resource_ldap_configuration_test.go

```diff
@@ -30,16 +30,23 @@ func TestAccLDAPConfiguration_withVerify_CACertificateComplete(t *testing.T) {
 		password           = os.Getenv("MONGODB_ATLAS_LDAP_PASSWORD")
 		port               = os.Getenv("MONGODB_ATLAS_LDAP_PORT")
 		caCertificate      = os.Getenv("MONGODB_ATLAS_LDAP_CA_CERTIFICATE")
-		projectID          = acc.ProjectIDExecution(t)
-		clusterName        = acc.RandomClusterName()
+		clusterInfo        = acc.GetClusterInfo(t, &acc.ClusterRequest{
+			CloudBackup: true,
+			ReplicationSpecs: []acc.ReplicationSpecRequest{
+				{Region: "US_EAST_2"},
+			},
+		})
+		projectID           = clusterInfo.ProjectID
```

Comment:
what about passing clusterInfo to configWithVerify so no need to create intermediate vars?

Reply:
Discussed offline